pennylane/qnode.py

```diff
@@ -552,13 +552,23 @@ def construct(self, args, kwargs):
             )
 
         for obj in self.qtape.operations + self.qtape.observables:
+

```

Comment:
I can't comment on it, but you may need to disable the codefactor warning above with a comment :)